src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
```

Comment:
you can move both types Todo and User into one file here called `index.ts` and just import types from here, you will have one file instead of two so it reduces codebase :)